egs/ptb/LM/local/train_bpe_model.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
```

Comment:
Also, can we replace it with a symlink to `egs/librispeech/ASR/local/train_bpe_model.py`?